### PR TITLE
EZP-31602: Moving to trash confirmation modal is not updated after su…

### PR DESCRIPTION
**Pull request by OstafinL**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/EZP-31602
| Related to | https://github.com/ezsystems/ezplatform-admin-ui/pull/1744
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/modules/sub-items/sub.items.module.js [hunk 1]
```diff
@@ -969,6 +962,8 @@ export default class SubItemsModule extends Component {
 
         const selectedPageLocationsIds = this.getPageSelectedLocationsIds();
 
+        this.updateTrashModal();
```
Comment (dew326) [inline src/modules/sub-items/sub.items.module.js]: Not sure it this should be fired every time when React is rerendering the list view

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/modules/sub-items/sub.items.module.js [hunk 2]
```diff
@@ -443,7 +443,7 @@ export default class SubItemsModule extends Component {
         this.updateTotalCountState(totalCount - movedLocations.length);
         this.deselectAllItems();
         this.discardActivePageItems();
-
+        this.updateTrashModal();
```
Comment (tischsoic) [inline src/modules/sub-items/sub.items.module.js:446]: Probably should be also added in `afterBulkDelete` method.

**Review by tischsoic — CHANGES_REQUESTED** [review]

**Review by tischsoic — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by kacper-wieczorek-ibexa — APPROVED** [review]
QA Approve - tested on 2.5
Tests on v3 after merge-up

**Comment by lserwatka** [comment]
@mnocon is it safe to merge it?

**Comment by mnocon** [comment]
@lserwatka the tests here failed because at that time there was an issue in kernel (now fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3097), same for https://github.com/ezsystems/ezplatform-admin-ui/pull/1744

I've restarted both jobs, but I expect both of them to be green

**Comment by lserwatka** [comment]
Sounds great, thank you for checking @mnocon 👍

**Comment by lserwatka** [comment]
You can merge it up.

**Pull request merged**